<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-Q8UPSCI" failures="0" tests="6" name="testNGDemo.TestDemo1" time="0.022" errors="1" timestamp="2020-07-03T00:45:06 IST" skipped="0">
  <testcase classname="testNGDemo.TestDemo1" name="testCase2" time="0.008"/>
  <system-out/>
  <testcase classname="testNGDemo.TestDemo1" name="signUp" time="0.000">
    <error message="
Parameter &#039;URL&#039; is required by @Test on method signUp but has not been marked @Optional or defined
in D:\JavaWorkspace\RahulShettyTraining\parameterisation.xml" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
Parameter 'URL' is required by @Test on method signUp but has not been marked @Optional or defined
in D:\JavaWorkspace\RahulShettyTraining\parameterisation.xml
at org.testng.internal.Parameters.createParams(Parameters.java:272)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:360)
at org.testng.internal.Parameters.createParameters(Parameters.java:704)
at org.testng.internal.Parameters.handleParameters(Parameters.java:879)
at org.testng.internal.Parameters.handleParameters(Parameters.java:744)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:783)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- signUp -->
  <system-out/>
  <testcase classname="testNGDemo.TestDemo1" name="testCase1" time="0.006"/>
  <system-out/>
  <testcase classname="testNGDemo.TestDemo1" name="testCase2" time="0.001"/>
  <system-out/>
  <testcase classname="testNGDemo.TestDemo1" name="signUp" time="0.005"/>
  <system-out/>
  <testcase classname="testNGDemo.TestDemo1" name="testCase1" time="0.002"/>
  <system-out/>
</testsuite> <!-- testNGDemo.TestDemo1 -->
